Adyen/Assets/Generated/LocalizationKey.swift

```diff
@@ -2,7 +2,7 @@
 // Copyright (c) 2021 Adyen N.V.
 //
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
-// This file is autogenerated. Please do not modify it.
```

Comment:
we need to do something with this line...

Reply:
yeah, I agree, I'll see what I can do